aspnetcore/blazor/host-and-deploy/server.md

```diff
@@ -62,91 +62,103 @@ Recommendations for global deployments to geographical data centers:
+:::moniker range=">= aspnetcore-8.0"
 
-* Set the option in `Program.cs`:
+<!-- UPDATE 9.0 Update section to only cross-link stateful
+                reconnect guidance after the feature is
+                supported with Azure SignalR Service. -->
 
-  ```csharp
-  builder.Services.AddSignalR().AddAzureSignalR(options =>
-  {
-      options.ServerStickyMode = 
-          Microsoft.Azure.SignalR.ServerStickyMode.Required;
-  });
-  ```
+> [!NOTE]
+> [Stateful reconnect](xref:signalr/configuration#configure-stateful-reconnect) (<xref:Microsoft.AspNetCore.SignalR.Client.HubConnectionBuilderHttpExtensions.WithStatefulReconnect%2A>) was released with .NET 8 but isn't currently supported for the Azure SignalR Service. For more information, see [Stateful Reconnect Support? (`Azure/azure-signalr` #1878)](https://github.com/Azure/azure-signalr/issues/1878).
 
-* Configure the option in `appsettings.json`:
+:::moniker-end
 
-  ```json
-  "Azure:SignalR:ServerStickyMode": "Required"
-  ```
+We recommend using WebSockets for Blazor apps deployed to Azure App Service. If [WebSockets](https://wikipedia.org/wiki/WebSocket) are disabled, Azure App Service simulates a real-time connection using HTTP Long Polling. HTTP Long Polling is noticeably slower than running with WebSockets enabled, which doesn't use polling to simulate a client-server connection.
 
-> [!NOTE]
-> The following error is thrown by an app that hasn't enabled session affinity:
->
-> > :::no-loc text="blazor.server.js:1 Uncaught (in promise) Error: Invocation canceled due to the underlying connection being closed.":::
+In the event that Long Polling must be used, you may need to configure the maximum poll interval (`MaxPollIntervalInSeconds`, default: 5 seconds, limit: 1-300 seconds), which defines the maximum poll interval allowed for Long Polling connections in the Azure SignalR Service if the service ever falls back from WebSockets to Long Polling. If the next poll request doesn't arrive within the maximum poll interval, the service closes the client connection. The service also cleans up connections when the cached waiting-to-write buffer size is greater than 1 MB for improved service performance.
 
-To provision the Azure SignalR Service for an app in Visual Studio:
+### Add the service as a dependency to a production deployment
 
-1. Create an Azure Apps publish profile in Visual Studio for the app.
-1. Add the **Azure SignalR Service** dependency to the profile. If the Azure subscription doesn't have a pre-existing Azure SignalR Service instance to assign to the app, select **Create a new Azure SignalR Service instance** to provision a new service instance.
-1. Publish the app to Azure.
+For guidance, see <xref:signalr/publish-to-azure-web-app>.
 
-Provisioning the Azure SignalR Service in Visual Studio automatically adds the SignalR connection string to the app service's configuration.
+### Use the service locally
```

Comment:
I don't think we really need to document here how to use the Azure SignalR Service locally during development with a Blazor app. I just don't think it's a common scenario for Blazor developers.

@mikekistler @BrennanConroy Is this scenario important for SignalR devs? Should this content maybe move to the https://review.learn.microsoft.com/aspnet/core/signalr/publish-to-azure-web-app doc?

Reply:
BTW @danroth27 ... This was my attempt to avoid cross-linking the chat sample, which has this, given your remarks about that sample requiring an update.

Reply:
> This was my attempt to avoid cross-linking the chat sample, which has this, given your remarks about that sample requiring an update.

Yup, I know. I don't think we need this content in the Blazor docs, but maybe it makes sense to add to the SignalR docs. Let's see what @mikekistler and @BrennanConroy say.

Reply:
I removed the section of local use of the service. In case the content that I placed is of use in the SignalR docs, here it is as a cut-'n-paste, and I can place it using this PR 👂 ...

> \#\#\# Use the service locally
>
> Add package reference to the app for \[\`Microsoft.Azure.SignalR\`](https://www.nuget.org/packages/Microsoft.Azure.SignalR).
> 
> \[!INCLUDE\[](~/includes/package-reference.md)]
> 
> Implement and configure the Azure SignalR Service in the app's service configuration of the `Program` file:
> 
> \`\`\`csharp
> builder.Services.AddSignalR().AddAzureSignalR();
> \`\`\`
> 
> Configure the following for the service in the app settings file (\`appsettings.json\`):
> 
> \* Session affinity (sticky sessions) with the \`ServerStickyMode\` key. Set the value to \`Required\`.
> \* Connection string (\`{CONNECTION STRING}\` placeholder in the following example). To avoid maintaining the connection string in plain text in the file, use the \[Secret Manager tool](xref:security/app-secrets) to supply the string.
> 
> In \`appsettings.json\`:
> 
> \`\`\`json
> "Azure": {
>   "SignalR": {
>     "Enabled": true,
>     "ServerStickyMode": "Required",
>     "ConnectionString": {CONNECTION STRING}
>   }
> }
> \`\`\`
> 
> Assign the Azure SignalR Service SDK as a hosting startup assembly. Edit \`Properties/launchSettings.json\`. Add an \`ASPNETCORE_HOSTINGSTARTUPASSEMBLIES\` entry to \`environmentVariables\` of the \`https\` profile with a value of \`Microsoft.Azure.SignalR\`:
> 
> \`\`\`json
> "environmentVariables": {
>     "ASPNETCORE_ENVIRONMENT": "Development",
>     "ASPNETCORE_HOSTINGSTARTUPASSEMBLIES": "Microsoft.Azure.SignalR"
> }
> \`\`\`